doc/source/whatsnew/v1.2.0.rst

```diff
@@ -462,6 +462,7 @@ Groupby/resample/rolling
 - Bug in :meth:`RollingGroupby.count` where a ``ValueError`` was raised when specifying the ``closed`` parameter (:issue:`35869`)
 - Bug in :meth:`DataFrame.groupby.rolling` returning wrong values with partial centered window (:issue:`36040`).
 - Bug in :meth:`DataFrameGroupBy.rolling` returned wrong values with timeaware window containing ``NaN``. Raises ``ValueError`` because windows are not monotonic now (:issue:`34617`)
+- Bug in :meth:`DataFrameGroupBy.quantile` and :meth:`DataFrame.resample().quantile()` raised ``TypeError`` when values to calculate quantile over where ``Timedelta`` (:issue:`29485`)
```

Comment:
@jorisvandenbossche Can we use a tilde to hide the full path? That is if we do :meth:\`~pandas.core.groupby.DataFrameGroupBy.quantile\` then I think it will only show the method at the end but still provide a link. Then could use the surrounding text to provide more context of which quantile method we're talking about.

Reply:
Indeed, that's yet another option ;) So as you say, since the link with only show `quantile()`, then the surrounding text needs to clarify what it is about, like "groupby `quantile()`" or "`quantile()` with groupby" or something alike. 

Personally I don't have a big preference for either solution (as long as we are not showing "(core.groupby.)DataFrameGroupBy" as we are doing now). I think a link text like `df.groupby(..).quantile()` is also quite clear in what it is about. Probably depends a bit on the full sentence which might fit best in which case.

Reply:
The last option @jorisvandenbossche gave:

```
:meth:`df.groupby(..).quantile() <pandas.core.groupby.DataFrameGroupBy.quantile>`
```

seems to be the best of all words to me.

Reply:
Yeah, I think thats probably the best solution. Is a bit more to write for us, but clearer to the reader.